index.html

```diff
@@ -0,0 +1,289 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task 1</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="body">
+        <div class="nav">
+            <ul class="navbar-nav">
+                <li class="nav-item">
+                    <a href="#"></a>
+                    <a href="#"></a>
+                    <a href="#"></a>
+                </li>
+            </ul>
+            <ul class="navbar-nav_bottom">
+                <li class="nav-item_bottom">
+                    <a href="#"></a>
+                    <a href="#"></a>
+                    <a href="#"></a>
+                </li>
+            </ul>
+        </div>
+        <div class="content">
+            <div class="Cards_Container">
```

Comment:
add max width to it, and make the width adjustable by using `width:100%` in its children instead of using fixed width